### PR TITLE
ansible: mask systemd-binfmt service on qemu-user-static install

### PR DESCRIPTION
**Pull request by theihor**

### CODE REVIEW

File: ansible/roles/qemu-user-static/tasks/main.yml [hunk 1]
```diff
@@ -23,6 +23,14 @@
     group: root
   register: qemu_user_static
 
+- name: Mask systemd-binfmt service
+  become: true
+  systemd:
+    name: systemd-binfmt
+    state: stopped
+    masked: yes
+  register: mask_systemd_binfmt
```
Comment (chantra) [inline ansible/roles/qemu-user-static/tasks/main.yml:32]: I don't think this is needed if you don't use it later: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#registering-variables

**Review by chantra — APPROVED** [review]
not a big deal, but you can probably remove that register keyword.

**Comment by chantra** [comment]
Also adding more details to the patch/PR description will help a potential future person to understand why the decision was made to mask that service.

File: ansible/roles/qemu-user-static/tasks/main.yml (hunk 1 above)
Comment (theihor) [reply ansible/roles/qemu-user-static/tasks/main.yml]: Removed.

**Comment by theihor** [comment]
> Also adding more details to the patch/PR description will help a potential future person to understand why the decision was made to mask that service.

Updated commit message. Please let me know if I missed anything important.

**Comment by chantra** [comment]
LGTM. Feel free to land whenever.

**Pull request merged**